Build failure: mtpaint

### Nixpkgs version

- Unstable (26.05)

### Steps to reproduce

`nix-shell -p mtpaint`, not sure what else to say here...

### Can Hydra reproduce this build failure?

Yes, Hydra can reproduce this build failure.

### Link to Hydra build job

https://hydra.nixos.org/build/322152600

### Relevant log output

```console
this derivation will be built:
  /nix/store/fvafzrvy18aslcbxxmz47pggydv86gml-mtPaint-3.50.12.drv
building '/nix/store/fvafzrvy18aslcbxxmz47pggydv86gml-mtPaint-3.50.12.drv'...
Running phase: unpackPhase
unpacking source archive /nix/store/132qsqk6svl53givab2fb71l1pcswsjm-source
source root is source
Running phase: patchPhase
Running phase: updateAutotoolsGnuConfigScriptsPhase
Running phase: configurePhase
patching script interpreter paths in ./configure
./configure: interpreter directive changed from "#!/bin/sh" to "/nix/store/2hjsch59amjs3nbgh7ahcfzm2bfwl8zi-bash-5.3p9/bin/sh"
configure flags: --prefix=/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12 gtk3 intl man


---------------------
mtPaint Configuration
---------------------

-------
General
-------

    Version             3.50.12
    System              GNU/Linux
    Toolkit             GTK+3
    Lists               GTK+2
    File Picker         mtpaint
    Colour Picker       mtpaint
    Animation Package   Gifsicle
    Use FreeType        YES
    Use CMS             LittleCMS2
    Icon set            icons1
    Internationalized   YES
    Multithreaded       YES

--------
Compiler
--------

    Optimizations       YES
    CFLAGS              -O2  -fcommon
    LDFLAGS             -s 
    Static libraries    NONE

----------
File Types
----------

    Use JPEG            YES
    Use JPEG 2000       OpenJPEG2
    Use TIFF            YES
    Use WebP            NO

------------
Installation
------------

    Binary install      /nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/bin
    Locale install      /nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale
    Locale program      /nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale
    Install man page    YES
    Man page install    /nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/man

Running phase: buildPhase
build flags: -j24 SHELL=/nix/store/2hjsch59amjs3nbgh7ahcfzm2bfwl8zi-bash-5.3p9/bin/bash
make -C src
make -C po
make -C doc
make[1]: Entering directory '/build/source/po'
for iii in es cs fr pt pt_BR de pl tr zh_TW sk zh_CN ja ru gl nl it sv tl hu kab; do echo -n "$iii = "; msgfmt -v -o $iii.mo $iii.po; done
make[1]: Entering directory '/build/source/doc'
make[1]: Nothing to be done for 'all'.
make[1]: Leaving directory '/build/source/doc'
es = make[1]: Entering directory '/build/source/src'
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=main -c -o main.o main.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=mainwindow -c -o mainwindow.o mainwindow.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=inifile -c -o inifile.o inifile.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=png -c -o png.o png.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=memory -c -o memory.o memory.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=canvas -c -o canvas.o canvas.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=otherwindow -c -o otherwindow.o otherwindow.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=mygtk -c -o mygtk.o mygtk.c
822 translated messages, 187 untranslated messages.
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=viewer -c -o viewer.o viewer.c
cs = gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=polygon -c -o polygon.o polygon.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=layer -c -o layer.o layer.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=info -c -o info.o info.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=wu -c -o wu.o wu.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=prefs -c -o prefs.o prefs.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=ani -c -o ani.o ani.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=mtlib -c -o mtlib.o mtlib.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=toolbar -c -o toolbar.o toolbar.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=channels -c -o channels.o channels.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=csel -c -o csel.o csel.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=shifter -c -o shifter.o shifter.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=spawn -c -o spawn.o spawn.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=font -c -o font.o font.c
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=fpick -c -o fpick.o fpick.c
858 translated messages, 151 untranslated messages.
fr = 985 translated messages, 24 untranslated messages.
pt = 190 translated messages, 819 untranslated messages.
pt_BR = 448 translated messages, 561 untranslated messages.
de = 933 translated messages, 76 untranslated messages.
pl = gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=icons -c -o icons.o icons.c
577 translated messages, 432 untranslated messages.
tr = 302 translated messages, 707 untranslated messages.
zh_TW = 616 translated messages, 9 fuzzy translations, 384 untranslated messages.
sk = 1008 translated messages.
zh_CN = 777 translated messages, 232 untranslated messages.
ja = 796 translated messages, 9 fuzzy translations, 204 untranslated messages.
ru = 987 translated messages, 1 fuzzy translation, 21 untranslated messages.
gl = 803 translated messages, 206 untranslated messages.
nl = 854 translated messages, 155 untranslated messages.
it = 831 translated messages, 178 untranslated messages.
sv = 742 translated messages, 267 untranslated messages.
tl = layer.c: In function 'load_layers':
layer.c:469:17: warning: ignoring return value of 'fgets' declared with attribute 'warn_unused_result' [-Wunused-result]
  469 |                 fgets(tin, 256, fp);
      |                 ^~~~~~~~~~~~~~~~~~~
layer.c:490:17: warning: ignoring return value of 'fgets' declared with attribute 'warn_unused_result' [-Wunused-result]
  490 |                 fgets(tin, 256, fp);
      |                 ^~~~~~~~~~~~~~~~~~~
fpick.c: In function 'fpick_scan_directory':
fpick.c:533:25: warning: ignoring return value of 'getcwd' declared with attribute 'warn_unused_result' [-Wunused-result]
  533 |                         getcwd(full_name, PATHBUF - 1);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
395 translated messages, 614 untranslated messages.
hu = 622 translated messages, 387 untranslated messages.
kab = 626 translated messages, 383 untranslated messages.
make[1]: Leaving directory '/build/source/po'
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=cpick -c -o cpick.o cpick.c
wu.c: In function 'Hist3d':
wu.c:72:13: warning: old-style function definition [-Wold-style-definition]
   72 | static void Hist3d(inbuf, vwt, vmr, vmg, vmb)   // build 3-D color histogram of counts, r/g/b, c^2
      |             ^~~~~~
wu.c: In function 'M3d':
wu.c:128:13: warning: old-style function definition [-Wold-style-definition]
  128 | static void M3d(vwt, vmr, vmg, vmb)     // compute cumulative moments.
      |             ^~~
wu.c: In function 'Vol':
wu.c:167:17: warning: old-style function definition [-Wold-style-definition]
  167 | static long int Vol(cube, mmt)                  // Compute sum over a box of any given statistic
      |                 ^~~
wu.c: In function 'Bottom':
wu.c:187:17: warning: old-style function definition [-Wold-style-definition]
  187 | static long int Bottom(cube, dir, mmt)
      |                 ^~~~~~
wu.c: In function 'Top':
wu.c:219:17: warning: old-style function definition [-Wold-style-definition]
  219 | static long int Top(cube, dir, pos, mmt)
      |                 ^~~
wu.c: In function 'Var':
wu.c:252:15: warning: old-style function definition [-Wold-style-definition]
  252 | static double Var(cube)
      |               ^~~
wu.c: In function 'Maximize':
wu.c:281:15: warning: old-style function definition [-Wold-style-definition]
  281 | static double Maximize(cube, dir, first, last, cut, whole_r, whole_g, whole_b, whole_w)
      |               ^~~~~~~~
gcc -Wall  -Wno-pointer-sign -Wno-int-to-pointer-cast -Wno-pointer-to-int-cast -Wno-misleading-indentation -Wno-stringop-truncation -Wformat-truncation=0 -Wformat-overflow=0 -DHAVE__SFA -DHAVE_MKDTEMP -DU_JP2=2 -DU_TIFF -DU_JPEG -DU_FREETYPE -DU_LCMS=2 -DU_NLS -DMT_LANG_DEST='"/nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12/share/locale"' -DU_THREADS -DU_FPICK_MTPAINT -DU_CPICK_MTPAINT -DMT_VERSION='"mtPaint 3.50.12"'  -I/nix/store/2rk026sb75l9balz9xg4qmscaxk8c03w-openjpeg-2.5.4-dev/include/openjpeg-2.5 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -pthread -I/nix/store/j74izjk0yig20z0xg0zp4mc6i43jriid-gtk+3-3.24.51-dev/include/gtk-3.0 -I/nix/store/xs67a36cap3rxv5rjx4vcy3mh7h2r5qm-at-spi2-core-2.58.3-dev/include/atk-1.0 -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include -I/nix/store/jhb15l57qmqxck5ddd1dh2ys729vc8gb-glib-2.86.3-dev/include/glib-2.0 -I/nix/store/1qm74vf93ik1xjrr9kl6qvjrklljlcqh-glib-2.86.3/lib/glib-2.0/include -I/nix/store/2pfjag759wflbzpvginva62kkn215c2j-cairo-1.18.4-dev/include/cairo -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include/freetype2 -I/nix/store/374f5mfap8y7i6rzc3qnq249ym1nxfdq-freetype-2.13.3-dev/include -I/nix/store/bfbb982lzrfzbvs5v9jwlnz0rbjvl8i9-gdk-pixbuf-2.44.5-dev/include/gdk-pixbuf-2.0 -I/nix/store/99l5iv6m6qzhq6mgfsn3zl74hil4wady-pango-1.57.0-dev/include/pango-1.0 -I/nix/store/x6y6khsf313j6sa6pjcv9ar0k4rkdph1-harfbuzz-12.3.0-dev/include/harfbuzz  -O2  -fcommon -frandom-seed=thread -c -o thread.o thread.c
spawn.c: In function 'spawn_process':
csel.c:102:22: error: static declaration of 'exp10' follows non-static declaration
  102 | static inline double exp10(double x)
      |                      ^~~~~
spawn.c:813:40: warning: ignoring return value of 'chdir' declared with attribute 'warn_unused_result' [-Wunused-result]
  813 |                         if (directory) chdir(directory);
      |                                        ^~~~~~~~~~~~~~~~
In file included from /nix/store/j8irrc0mpx029dw0rmadsjylg7h31ync-glibc-2.42-51-dev/include/features.h:524,
                 from /nix/store/j8irrc0mpx029dw0rmadsjylg7h31ync-glibc-2.42-51-dev/include/libintl.h:23,
                 from global.h:22,
                 from csel.c:20:
/nix/store/j8irrc0mpx029dw0rmadsjylg7h31ync-glibc-2.42-51-dev/include/bits/mathcalls.h:136:1: note: previous declaration of 'exp10' with type 'double(double)'
  136 | __MATHCALL_VEC (exp10,, (_Mdouble_ __x));
      | ^~~~~~~~~~~~~~
spawn.c:824:25: warning: ignoring return value of 'write' declared with attribute 'warn_unused_result' [-Wunused-result]
  824 |                         write(fds[1], &err, sizeof(err));
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:20: csel.o] Error 1
make[1]: *** Waiting for unfinished jobs....
png.c: In function 'load_png':
png.c:759:56: error: passing argument 3 of 'png_set_read_user_chunk_fn' from incompatible pointer type [-Wincompatible-pointer-types]
  759 |         else png_set_read_user_chunk_fn(png_ptr, NULL, buggy_libpng_handler);
      |                                                        ^~~~~~~~~~~~~~~~~~~~
      |                                                        |
      |                                                        int (*)(void)
In file included from /nix/store/678ygbangkyc4yb5xm7qb4psfpk3vw4c-libpng-apng-1.6.55-dev/include/png.h:337,
                 from png.c:27:
/nix/store/678ygbangkyc4yb5xm7qb4psfpk3vw4c-libpng-apng-1.6.55-dev/include/png.h:1765:1: note: expected 'png_user_chunk_ptr' {aka 'int (*)(struct png_struct_def *, struct png_unknown_chunk_t *)'} but argument is of type 'int (*)(void)'
 1765 | PNG_EXPORT(88, void, png_set_read_user_chunk_fn,
      | ^~~~~~~~~~
png.c:666:12: note: 'buggy_libpng_handler' declared here
  666 | static int buggy_libpng_handler()
      |            ^~~~~~~~~~~~~~~~~~~~
/nix/store/678ygbangkyc4yb5xm7qb4psfpk3vw4c-libpng-apng-1.6.55-dev/include/png.h:845:9: note: 'png_user_chunk_ptr' declared here
  845 | typedef PNG_CALLBACK(int, *png_user_chunk_ptr,
      |         ^~~~~~~~~~~~
mygtk.c: In function 'resolve_path':
mygtk.c:5584:17: warning: ignoring return value of 'getcwd' declared with attribute 'warn_unused_result' [-Wunused-result]
 5584 |                 getcwd(wbuf, PATHBUF - 1);
      |                 ^~~~~~~~~~~~~~~~~~~~~~~~~
png.c: In function 'load_xpm':
png.c:5765:16: warning: ignoring return value of 'fscanf' declared with attribute 'warn_unused_result' [-Wunused-result]
 5765 |         j = 0; fscanf(fp, " /* XPM */%n", &j);
      |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
png.c: In function 'load_txtpal':
png.c:9435:25: warning: ignoring return value of 'fscanf' declared with attribute 'warn_unused_result' [-Wunused-result]
 9435 |                         fscanf(fp, "%i,%i,%i\n", rgb + 0, rgb + 1, rgb + 2);
      |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
make[1]: *** [Makefile:20: png.o] Error 1
make[1]: Leaving directory '/build/source/src'
make: *** [Makefile:8: src] Error 2
error: Cannot build '/nix/store/fvafzrvy18aslcbxxmz47pggydv86gml-mtPaint-3.50.12.drv'.
       Reason: builder failed with exit code 2.
       Output paths:
         /nix/store/8r43h1x20yil7aiqng8rgjnls2igmnfa-mtPaint-3.50.12
       Last 25 log lines:
       >                  from png.c:27:
       > /nix/store/678ygbangkyc4yb5xm7qb4psfpk3vw4c-libpng-apng-1.6.55-dev/include/png.h:1765:1: note: expected 'png_user_chunk_ptr' {aka 'int (*)(struct png_struct_def *, struct png_unknown_chunk_t *)'} but argument is of type 'int (*)(void)'
       >  1765 | PNG_EXPORT(88, void, png_set_read_user_chunk_fn,
       >       | ^~~~~~~~~~
       > png.c:666:12: note: 'buggy_libpng_handler' declared here
       >   666 | static int buggy_libpng_handler()
       >       |            ^~~~~~~~~~~~~~~~~~~~
       > /nix/store/678ygbangkyc4yb5xm7qb4psfpk3vw4c-libpng-apng-1.6.55-dev/include/png.h:845:9: note: 'png_user_chunk_ptr' declared here
       >   845 | typedef PNG_CALLBACK(int, *png_user_chunk_ptr,
       >       |         ^~~~~~~~~~~~
       > mygtk.c: In function 'resolve_path':
       > mygtk.c:5584:17: warning: ignoring return value of 'getcwd' declared with attribute 'warn_unused_result' [-Wunused-result]
       >  5584 |                 getcwd(wbuf, PATHBUF - 1);
       >       |                 ^~~~~~~~~~~~~~~~~~~~~~~~~
       > png.c: In function 'load_xpm':
       > png.c:5765:16: warning: ignoring return value of 'fscanf' declared with attribute 'warn_unused_result' [-Wunused-result]
       >  5765 |         j = 0; fscanf(fp, " /* XPM */%n", &j);
       >       |                ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > png.c: In function 'load_txtpal':
       > png.c:9435:25: warning: ignoring return value of 'fscanf' declared with attribute 'warn_unused_result' [-Wunused-result]
       >  9435 |                         fscanf(fp, "%i,%i,%i\n", rgb + 0, rgb + 1, rgb + 2);
       >       |                         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
       > make[1]: *** [Makefile:20: png.o] Error 1
       > make[1]: Leaving directory '/build/source/src'
       > make: *** [Makefile:8: src] Error 2
       For full logs, run:
         nix log /nix/store/fvafzrvy18aslcbxxmz47pggydv86gml-mtPaint-3.50.12.drv
```

### Additional context

_No response_

### System metadata

 - system: `"x86_64-linux"`
 - host os: `Linux 6.19.5, NixOS, 26.05 (Yarara), 26.05.20260302.cf59864`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.31.3`
 - channels(root): `"nixos-25.11, nixos-hardware"`
 - nixpkgs: `/nix/store/s08d9f5avsqs69r1y2gyivxgfwv4a88c-source`



### Notify maintainers

None :(

---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+label%3A%220.kind%3A+build+failure%22).
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/by-name/mt/mtpaint/package.nix
+++ b/pkgs/by-name/mt/mtpaint/package.nix
@@ -17,13 +17,13 @@
 
 stdenv.mkDerivation {
   pname = "mtPaint";
-  version = "3.50.12";
+  version = "3.50.14";
 
   src = fetchFromGitHub {
     owner = "wjaguar";
     repo = "mtPaint";
-    rev = "7cae5d663ed835a365d89a535536c39e18862a83";
-    hash = "sha256-W/MQZ1WqoVMzyEd60rbvA8yieesDc/xfKqbYGZumi2U=";
+    rev = "8304376e8861a8a603371b0f188db30f9cafdc17";
+    hash = "sha256-dyBbzEjdgMPlPnjFlJoZOh5qjx/qY94F28jEr2ihLQE=";
   };
 
   strictDeps = true;

